src/pages/404.astro

```diff
@@ -0,0 +1,39 @@
+---
+import BaseLayout from '../layouts/BaseLayout.astro';
+---
+
+<BaseLayout>
+  <div class="px-16 py-8 flex flex-col items-center space-y-6 h-full justify-center">
+    <div class="flex flex-col items-center space-y-2">
+      <p class="text-content-accent type-body-2 font-bold">
+        404
+      </p>
+      <h1 class="type-headline-2 text-gray-900">
```

Comment:
I think this should be a custom Centrapay colour token next time 👀

Reply:
Do you have a suggestion for which one to use? Looks like there's a few cases of `text-gray-*` so maybe we need some new classes to capture these